src/stories/containers/RecognizedDelegates/KeyStats/KeyDescriptionCards/PercentageTotalCard.tsx

```diff
@@ -0,0 +1,42 @@
+import styled from '@emotion/styled';
+import React from 'react';
+import KeyStatsCard from '../../GenericDelegateCard';
+import { DescriptionDelegates } from './TotalRecognizedDelegatesCard';
+
+interface Props {
+  percent: string;
+}
+
+const PercentageTotalCard: React.FC<Props> = ({ percent }) => (
+  <ExtendedGenericDelegate>
+    <Annual>{percent}</Annual>
+    <DescriptionDelegatesExtended>{'Percentage of Total DAO Expense Nov 2021 - Jun 2023'}</DescriptionDelegatesExtended>
+  </ExtendedGenericDelegate>
+);
+
+export default PercentageTotalCard;
+
+const ExtendedGenericDelegate = styled(KeyStatsCard)({
+  padding: '8px 18.5px',
+  display: 'flex',
+  flexDirection: 'column',
+  alignItems: 'center',
```

Comment:
maybe would be a good idea to move that code to the generic card as it looks like it is used every time you extended the generic component

Reply:
Not in this case, this card is used in other part , and the distribution is different